lib/l10n/intl_en.arb

```diff
@@ -17,5 +17,21 @@
   "notEqualErrorText": "This field value must not be equal to {value}.",
   "numericErrorText": "Value must be numeric.",
   "requiredErrorText": "This field cannot be empty.",
-  "urlErrorText": "This field requires a valid URL address."
+  "urlErrorText": "This field requires a valid URL address.",
+  "phoneErrorText": "This field requires a valid phone number.",
+  "creditCardExpirationDateErrorText": "This field requires a valid expiration date.",
+  "creditCardExpiredErrorText": "This credit card has expired.",
+  "creditCardCVCErrorText": "This field requires a valid CVC code.",
+  "colorCodeErrorText": "Value should be a valid {colorCode} color code.",
+  "uppercaseErrorText": "Value must be uppercase.",
+  "lowercaseErrorText": "Value must be lowercase.",
+  "fileExtensionErrorText": "File extension must be {extensions}",
+  "fileSizeErrorText": "File size must be less than {maxSize} while it is {fileSize}",
+  "dateRangeErrorText": "Date must be in range {min} - {max}",
+  "mustBeTrueErrorText": "This field must be true.",
+  "mustBeFalseErrorText": "This field must be false.",
+  "containsSpecialCharErrorText": "Value must contain at least {min} special characters.",
+  "containsUppercaseCharErrorText": "Value must contain at least {min} uppercase characters.",
+  "containsLowercaseCharErrorText": "Value must contain at least {min} lowercase characters.",
+  "containsNumberErrorText": "Value must contain at least {min} numbers."
```

Comment:
I know that is a hard work, but ideally should translate all error messages on all languages support. Or where available and more popular/used.